nflgame/game.py

```diff
@@ -114,9 +114,23 @@ def total_seconds(self):
         """
         return self.seconds + self.minutes * 60
 
-    def __cmp__(self, other):
-        a, b = (self.minutes, self.seconds), (other.minutes, other.seconds)
-        return cmp(a, b)
+    def __lt__(self, other):
+        return self.total_seconds() < other.total_seconds()
+
+    def __gt__(self, other):
```

Comment:
__cmp__ is no longer supported in Python 3 so had to add all the operators.

Reply:
@functools.total_ordering  will be very useful here.